gems/smithy/spec/spec_helper.rb

```diff
@@ -85,21 +77,14 @@ def load_model(modules, options)
 
     def create_plan(modules, model, type, options)
       plan_options = {
-        gem_name: options.fetch(:gem_name, Smithy::Util::Namespace.gem_name_from_namespaces(modules)),
-        gem_version: options.fetch(:gem_version, '1.0.0'),
-        destination_root: options.fetch(:destination_root, Dir.mktmpdir)
+        module_name: modules.join('::'),
+        gem_version: options.fetch(:gem_version, '0.1.0'),
+        destination_root: options.fetch(:destination_root, Dir.mktmpdir),
+        quiet: ENV.fetch('SMITHY_RUBY_DEBUG', 'false') == 'false'
```

Comment:
It feels a little weird for the ENV to not match the name.

Reply:
You mean SMITHY_RUBY_QUIET? I think that's less descriptive about what it does.. I'll think about it.

Reply:
We could also rename the cli parameter/plan option.  I personally think `SMITHY_RUBY_QUIET` is more descriptive than debug though, but I do think the env name, cli parameter and plan option shoudl all match.